a = 1
del a
print(a)  # This will raise a NameError since 'a' has been deleted
